### PR TITLE
R

### PR DESCRIPTION
**Pull request by Rogerrox**
log in page

### CODE REVIEW

File: public/pages/login.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                
+                <p class="history"> Find A Musician is a website dedicated to connecting musicians with other 
+                    like-minded musicians for collaboration and project assistance. Our platform allows musicians 
+                    to easily create an account and make posts asking for a helping hand or collaboration from other musicians.
+                    Whether you're looking for a guitarist for your band or a vocalist for your newest song our site offers a 
+                    user-friendly interface to help you find the right fit. Join our community of passionate and creative musicians 
+                    today and take your musical projects to the next level. 
+                                                                                       
+                </p>
+            </div>
```
Comment (ashleytrollinger) [inline public/pages/login.html]: All of this content is for the home page. This page just needs two <input = "text"> for the password and username so the user can log in to their account. Look at page 9 of the wireframe for a visual.

File: public/pages/login.html [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Find My Musician.com</title>
+</head>
+
+<body>
+    <header>
+        <div class="container">
+        <h1>Find My Musician.com </h1>
+        <nav>
+            <ul>
```
Comment (ashleytrollinger) [inline public/pages/login.html]: you can copy the header from the index.html so they all are they same and have the same ids.

File: public/pages/login.html [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+                <li>
+                    <a href="email-login">Log In</a>
```
Comment (ashleytrollinger) [inline public/pages/login.html]: These are links not buttons. Look at the index.html for reference

File: public/pages/login.html [hunk 4]
```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="../style/style.css">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (ashleytrollinger) [inline public/pages/login.html]: Make sure to put the style in the head as well

**Review by ashleytrollinger — CHANGES_REQUESTED** [review]

File: public/pages/login.html (hunk 4 above)
Comment (Rogerrox) [reply public/pages/login.html]: I believe style is in there already, let me know if is not right 
<link rel="stylesheet" href="../style/style.css">

**Review by ashleytrollinger — APPROVED** [review]

**Review by QDatcher — APPROVED** [review]

**Review by QDatcher — APPROVED** [review]

**Pull request merged**